### ISSUE TITLE
Repair analyser_merge_defibrillators_FR.py

### ISSUE THREAD

**Issue by johanricher** [osm-fr/osmose-backend#2268]
The French national database for defibrillators is published in JSON on data.gouv.fr: https://www.data.gouv.fr/fr/datasets/geodae-base-nationale-des-defibrillateurs/#/resource-86ea48a0-dd94-4a23-b71c-80d3041d7db2.

In the Osmose project, [`analysers/analyser_merge_defibrillators_FR.py`](https://github.com/osm-fr/osmose-backend/blob/master/analysers/analyser_merge_defibrillators_FR.py) was made to help reconcile and merge this data into OSM.

However, the structure of the JSON file seems to have changed since then. Geo coordinates are not in `geom_x`, `geom_y` fields but rather `c_lat_coor1` and `c_long_coor1`.

Osmose integration of defibrillators should be repaired to help add this data into OSM

**Comment by frodrigo** [comment]
It not looks broken to me. The analyser use GeoJSON format, not JSON.

https://osmose.openstreetmap.fr/en/issues/open?item=8370

**Comment by johanricher** [comment]
`analysers/analyser_merge_defibrillators_FR.py` references the id (`86ea48a0-dd94-4a23-b71c-80d3041d7db2`) of the `geodae.json` file published on data.gouv.fr (which is the only one on the dataset page, the other is a CSV file).

As far as I can tell, there are more than 132k unique defibrillators in `geodae.json`. I doesn't seem like they are all in Osmose. For example, in `geodae.json` there are 6 defibrillators for the commune of Chaufailles (filtering the `c_com_cp` field with the `71170` value), whereas Osmose shows nothing for [this area](https://osmose.openstreetmap.fr/fr/map/#item=8370&loc=14/46.20873/4.34184).

Of course, it's extremely possible that my comprehension of how Osmose works is very bad. :) In any case I'll learn something, thank you for your help!

**Comment by frodrigo** [comment]
Right, results are provided by this one `analyser_merge_defibrillators_FR_aedmap.py`.

The other one reports nothing

https://osmose.openstreetmap.fr/en/issues/open?item=8370&class=120
